### PR TITLE
Fix: Sign-in advanced option focus UI fix [CRNS - 363]

### PR DESCRIPTION
**Pull request by khushal87**
## 🎯 Goal

This PR fixes the sign-in advanced option UI in the SampleApp.

## 🛠 Implementation details

The input focus border color didn't work previously so now on focusing an input the border color of the text input is changed to `accent_blue`.

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                 
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [x] Commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec
- [ ] New code is covered by tests
- [x] Screenshots added for visual changes
- [ ] Documentation is updated

### CODE REVIEW

**Review by vishalnarkhede — APPROVED** [review]

File: examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx [hunk 1]
```diff
@@ -64,15 +64,27 @@ export const LabeledTextInput: React.FC<LabeledTextInputProps> = ({
...
+
+  const onFocus = () => {
+    setBorderColor(accent_blue);
+  };
+
+  const onBlur = () => {
+    setBorderColor(white_smoke);
+  };
+
   return (
```
Comment (madsroskar) [inline examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx:80]:
```suggestion
  };
  
  const isEmpty = value === undefined;

  return (
```

File: examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx [hunk 2]
```diff
@@ -102,13 +114,18 @@ export const LabeledTextInput: React.FC<LabeledTextInputProps> = ({
...
         placeholder={label}
         placeholderTextColor={grey}
         returnKeyType='next'
         style={[
           styles.input,
-          { color: black, fontWeight: value ? undefined : '500' }, // design team wanted placeholder fontWeight of 500
+          {
+            color: black,
+            fontWeight: value ? undefined : '500',
+          }, // design team wanted placeholder fontWeight of 500
```
Comment (madsroskar) [inline examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx:128]:
```suggestion
            color: black,
            fontWeight: isEmpty && '500',
          }, // design team wanted placeholder fontWeight of 500
```

**Review by madsroskar — APPROVED** [review]
Looks good to me, just added a suggestion to make a little change to a check that's here from before

File: examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx (hunk 2 above)
Comment (khushal87) [reply examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx:128]: This gives an error:

` Types of property 'fontWeight' are incompatible.`

Comment (madsroskar) [reply examples/SampleApp/src/screens/AdvancedUserSelectorScreen.tsx:128]: `isEmpty ? 500 : 'normal'` should work

**Pull request merged**